benchmark/agbenchmark/app.py

```diff
@@ -56,7 +56,8 @@
     except ValidationError as e:
         if logging.getLogger().level == logging.DEBUG:
             logger.warning(f"Spec file {challenge_relpath} failed to load:\n{e}")
-        logger.debug(f"Invalid challenge spec: {challenge_spec_file.read_text()}")
+        with open(challenge_spec_file, 'r', encoding='utf-8') as f:
+            logger.debug(f"Invalid challenge spec: {f.read_text()}")
```

Comment:
`encoding` can be passed to `read_text` so you don't need to use `with`:
```suggestion
        logger.debug(f"Invalid challenge spec: {challenge_spec_file.read_text(encoding='utf-8')}")
```